### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by kaikai1024**
Check the https://github.com/facebook/create-react-app#yarn

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -64,14 +64,10 @@
...
 
-```shell
-$ yarn global add create-react-app
-```
-
 初始化项目
 
 ```shell
-$ create-react-app first-forever && cd first-forever
+$ yarn create react-app first-forever && cd first-forever
```
Comment (Keith-CY) [inline README-CN.md:70]: Do you mean `npx create-react-app first-forever && cd first-forever` ?

Comment (rainchen) [reply README-CN.md:70]: is create-react-app deprecated?

Comment (kaikai1024) [reply README-CN.md:70]: I can't use this command on my environment.
Check this: https://github.com/facebook/create-react-app#yarn

@Keith-CY https://github.com/facebook/create-react-app#npx

I did not try this. `yarn` is ok.

Comment (Keith-CY) [reply README-CN.md:70]: > I can't use this command on my environment.
> Check this: https://github.com/facebook/create-react-app#yarn
> 
> @Keith-CY https://github.com/facebook/create-react-app#npx
> 
> I did not try this. `yarn` is ok.

Fine, `yarn` is more general.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**